Aptoide/aptoide-uploader
# [Feature/APP-84] Auto Upload

**What does this PR do?**

   Adds autoupload feature to the uploader. 
   - Tooltip on first time app is launched
   - Settings screen 
   - Auto upload screen with list to select the apps
   Service works in the background, when an app is updated and is from the selected list, the upload will start. 

**Where should the reviewer start?**

- InstallManager.kt
- AutoUploadSelectsManager.kt
- InstalledAppsManager.java
- AutoUploadPresenter.java
- SettingsPresenter.java

**How should this be manually tested?**

- Install an older version of an app from the Aptoide store
- Open Uploader and navigate to the autoupload (login -> myapps -> settings -> autoupload)
- Select the app that was installed. 
- Go back to vanilla and select a version of the app to update to.
- Check the notification of the upload after the new version was installed.

- Check on the seetings view, if the selected app image is shown.
- Select more apps (if 3 or less are selected, all images will show. if more than 3 are selected, 3 of them will show and the rest will be a number)

**What are the relevant tickets?**

  https://aptoide.atlassian.net/browse/APP-84

**Code Review Checklist**
